server/conf/application.dev-browser-tests.conf

```diff
@@ -0,0 +1,9 @@
+# Configuration for the dev browser test CiviForm server.
+include "application.dev.conf"
+
+# The dev server turns somethings on by default so
```

Comment:
This statement is going to be hard to enforce due to the fact that the configuration inherits from `application.dev.conf`.

What about having this inherit from `application.conf` and forking the current values? There's opportunity for drift there, but less surprise about things "being enabled by default" in dev.

Reply:
I thought about that, but was a little worried it be to much change from the status quo,  as well the naming wasn't clear.

If we made the dev server the one with a specific config (EG setting the feature flags for now) , I wasn't quite sure what to name that.  thoughts?

application.dev.conf -> application.dev_base.conf

Then inherit that in both:
- application.dev_server.conf
- application.dev_browser_tests_server.conf (effectively empty for now)

??

Reply:
Having an application.dev_base.conf would only exist in order to share the few common variables. In my opinion, the fact that they're mostly the same doesn't necessarily mean we should share them.

It feels like the reason the naming is challenging is because we're trying to build a relationship between the two conceptually that doesn't necessarily exist.

Reply:
I think you're saying we should duplicate the configuration rather than inherit, is that the case?

In reality they are the same though because their intent is to start the server in the same manner especially wrt to networking, fake components etc. the relationship is they're both trying to run a "dev configured" server environment, one for unit tests one for browser tests.

So duplicating feels like it could be error prone as most changes would have to be done twice.

Reply:
Fine to leave as-is if you'd like. I just have a sense that they happen to be the same right now, even though we might have different expectations of them as time goes on. When that time comes, we could revisit whether forking or duplicating makes more sense.

Reply:
I def like the idea of having them both branch off of the common thing, more like your suggestion I think, rather than assuming one is the "default", but not sure what naming won't be confusing....